samples/apps/smallbank/app/smallbank.cpp

```diff
@@ -23,6 +24,8 @@ namespace ccfapp
     static constexpr auto SMALL_BANKING_WRITE_CHECK = "SmallBank_write_check";
   };
 
+  static constexpr auto expected = http::headervalues::contenttype::TEXT;
```

Comment:
application/octet-stream would be more accurate, since this is arbitrary binary content rather than text. Not that it will have any functional or performance impact.